recipes/rnabob/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: rnabob
+  version: "2.2.1"
+
+source:
+  url: http://eddylab.org/software/rnabob/rnabob.tar.gz
+  patches:
+    - manlocation.patch
+
+requirements:
+  build:
+
+  run:
+  
+test:
+  commands:
+    - rnabob -h
+
+about:
+  home: http://selab.janelia.org
+  license: University of Toronto
```

Comment:
No, it isn't. This was just temporary. Can I leave it blank since there is a file?

The following is the part from the COPYING file that explains the license.

> Most of the code in patsearch.c is derived from copyrighted but freely
> distributable code of Henry Spencer's. patsearch.c carries Spencer's
> copyright notice for the University of Toronto in addition to my own.
> 
> Much of the code in sqio.c is derived from the freely distributable
> "readseq" library by Don Gilbert. sqio.c carries Gilbert's copyright
> notice in addition to my own.
> 
> Certain functions in sre_math.c are from freely distributable third
> party sources, and are credited and copyrighted as marked in the
> sre_math.c source.